pimd/pim_cmd.c

```diff
@@ -3118,13 +3118,17 @@ static void pim_show_group_rp_mappings_info(struct pim_instance *pim,
 	json_object *json = NULL;
 	json_object *json_group = NULL;
 	json_object *json_row = NULL;
+	struct prefix bsr;
 
-	if (pim->global_scope.current_bsr.s_addr == INADDR_ANY)
+	if (pim_addr_is_any(pim->global_scope.current_bsr))
```

Comment:
`bsr_str` is overwritten again directly below - what is the point of the strlcpy here?